docs/dev/python/table-api-users-guide/table_environment.md

```diff
@@ -0,0 +1,838 @@
+        the result of the given `Table` object or multi-sinks plan.
+        If you use the "insert_into" or "sql_update" method to emit data to multiple sinks, you can use this
+        method to get the plan.
+        It can be replaced by "explain_sql", `Table`.explain or `StatementSet`.explain().
+      </td>
+      <td class="text-center">
+        <a href="{{ site.pythondocs_baseurl }}/api/python/pyflink.table.html#pyflink.table.TableEnvironment.explain">link</a>
+      </td>
+    </tr>
+    <tr>
+      <td>
+        <strong>execute(job_name)</strong>
+      </td>
+      <td>
+        Triggers the program execution. The environment will execute all parts of the program.
+        If you use the "insert_into" or "sql_update" method to emit data to sinks, you can use this
+        method trigger the program execution.
+        This method will block the client program until the job is finished/canceled/failed.
+      </td>
+      <td class="text-center">
+        <a href="{{ site.pythondocs_baseurl }}/api/python/pyflink.table.html#pyflink.table.TableEnvironment.execute">link</a>
+      </td>
+    </tr>
+  </tbody>
+</table>
+
+### Create/Drop User Defined Functions
+
+These APIs are used to register UDFs or remove the registered UDFs. 
+Note that the API `execute_sql` can also be used to register/remove UDFs.
+For more introduction about different kinds of UDF, please refer to [User Defined Functions]({% link dev/table/functions/index.md %}).
+
+<table class="table table-bordered">
+  <thead>
+    <tr>
+      <th class="text-left" style="width: 20%">APIs</th>
+      <th class="text-center">Description</th>
+      <th class="text-center" style="width: 10%">Docs</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td>
+        <strong>create_temporary_function(path, function)</strong>
+      </td>
+      <td>
+        Registers a python user defined function class as a temporary catalog function.
+      </td>
+      <td class="text-center">
+        <a href="{{ site.pythondocs_baseurl }}/api/python/pyflink.table.html#pyflink.table.TableEnvironment.create_temporary_function">link</a>
+      </td>
+    </tr>
+    <tr>
+      <td>
+        <strong>create_temporary_system_function(name, function)</strong>
+      </td>
+      <td>
+        Registers a python user defined function class as a temporary system function.
+        If the name of a temporary system function is the same as a temporary catalog function,
+        the temporary system function takes precedence.
+      </td>
+      <td class="text-center">
+        <a href="{{ site.pythondocs_baseurl }}/api/python/pyflink.table.html#pyflink.table.TableEnvironment.create_temporary_system_function">link</a>
+      </td>
+    </tr>
+    <tr>
+      <td>
+        <strong>create_java_function(path, function_class_name, ignore_if_exists=None)</strong>
+      </td>
+      <td>
+        Registers a java user defined function class as a catalog function in the given path.
```

Comment:
Should we use Java instead of java and Python instead of python throughout the doc?